ansible_collections/arista/avd/molecule/eos_designs_unit_tests/custom_modules/custom_ip_addressing.py

```diff
@@ -8,107 +8,116 @@
+        return int(self._hostvars.get("ip_offset_10", 0))
 
     @cached_property
-    def _custom_ip_offset_10_subnets(self):
+    def _custom_ip_offset_10_subnets(self) -> int:
         """
         The jinja code did a blind addition of 10 to the resulting IP even for subnets.
-        Here we divide the offset with two, since we are calculating /31 subnets more intelligently
+
+        Here we divide the offset with two, since we are calculating /31 subnets more intelligently.
         """
         return int(self._custom_ip_offset_10 / 2)
 
     @cached_property
-    def _custom_ip_offset_20(self):
-        return self._hostvars.get("ip_offset_20", 0)
+    def _custom_ip_offset_20(self) -> int:
+        return int(self._hostvars.get("ip_offset_20", 0))
 
     @cached_property
-    def _custom_ip_offset_20_subnets(self):
+    def _custom_ip_offset_20_subnets(self) -> int:
         """
         The jinja code did a blind addition of 20 to the resulting IP even for subnets.
-        Here we divide the offset with two, since we are calculating /31 subnets more intelligently
+
+        Here we divide the offset with two, since we are calculating /31 subnets more intelligently.
         """
         return int(self._custom_ip_offset_20 / 2)
 
     def mlag_ibgp_peering_ip_primary(self, mlag_ibgp_peering_ipv4_pool: str) -> str:
         """
-        Implementation of custom code similar to jinja:
+        Implementation of custom code similar to jinja.
+
         {{ vrf.mlag_ibgp_peering_ipv4_pool | ansible.utils.ipaddr('subnet') |
-           ansible.utils.ipmath((switch.mlag_switch_ids.primary - 1) * 2 + ip_offset_10) }}
+           ansible.utils.ipmath((switch.mlag_switch_ids.primary - 1) * 2 + ip_offset_10) }}.
         """
         offset = self._mlag_primary_id - 1 + self._custom_ip_offset_10_subnets
         return self._ip(mlag_ibgp_peering_ipv4_pool, 31, offset, 0)
 
     def mlag_ibgp_peering_ip_secondary(self, mlag_ibgp_peering_ipv4_pool: str) -> str:
         """
-        Implementation of custom code similar to jinja:
+        Implementation of custom code similar to jinja.
+
         {{ vrf.mlag_ibgp_peering_ipv4_pool | ansible.utils.ipaddr('subnet') |
-           ansible.utils.ipmath(((switch.mlag_switch_ids.primary - 1) * 2) + 1 + ip_offset_10) }}
+           ansible.utils.ipmath(((switch.mlag_switch_ids.primary - 1) * 2) + 1 + ip_offset_10) }}.
         """
         offset = self._mlag_primary_id - 1 + self._custom_ip_offset_10_subnets
         return self._ip(mlag_ibgp_peering_ipv4_pool, 31, offset, 1)
 
     def mlag_ip_primary(self) -> str:
         """
-        Implementation of custom code similar to jinja:
+        Implementation of custom code similar to jinja.
+
         {{ switch_data.combined.mlag_peer_ipv4_pool | ansible.utils.ipaddr('network') |
-           ansible.utils.ipmath((mlag_primary_id - 1) * 2 + ip_offset_10) }}
+           ansible.utils.ipmath((mlag_primary_id - 1) * 2 + ip_offset_10) }}.
```

Comment:
same not sure if putting it in the Examples section can help?

Reply:
They got added by some autofix. I will remove as I find them